packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/initializingsdksession.md

```diff
@@ -1,26 +1,35 @@
 
-This example creates the client in the viewmodel `launch` coroutine. Start by calling `IdxClient.start()` and on success, request the first response by calling `resume()` on the result.
+This example creates the flow in the viewmodel `launch` coroutine. Start by calling `CredentialBootstrap.oidcClient.createInteractionCodeFlow` and on success, request the first response by calling `resume()` on the result.
```

Comment:
No need for parentheses after method name in Kotlin docs (as per style guide)

```suggestion
This example creates the flow in the viewmodel `launch` coroutine. Start by calling `CredentialBootstrap.oidcClient.createInteractionCodeFlow` and on success, request the first response by calling `resume` on the result.
```

Reply:
Fixed... the dangers of an iOS person cleaning up pros ;-).

It's in the next commit.